src/device.cpp

```diff
@@ -309,9 +309,15 @@ std::vector<rs2_format> device::map_supported_color_formats(rs2_format source_fo
     std::vector<rs2_format> target_formats = { RS2_FORMAT_RGB8, RS2_FORMAT_RGBA8, RS2_FORMAT_BGR8, RS2_FORMAT_BGRA8 };
     switch (source_format)
     {
+    case RS2_FORMAT_M420:
+        target_formats.push_back(RS2_FORMAT_M420);
+        target_formats.push_back(RS2_FORMAT_Y16);
+        target_formats.push_back(RS2_FORMAT_Y8);
+        break;
     case RS2_FORMAT_YUYV:
         target_formats.push_back(RS2_FORMAT_YUYV);
         target_formats.push_back(RS2_FORMAT_Y16);
+        target_formats.push_back(RS2_FORMAT_Y8);
```

Comment:
Can you explain this change?

Reply:
It permits to stream the YUYV as Y8 - it helped me when working on M420 format, in order to compare.
The conversion of YUYV to Y8 was already implemented, but not enabled to the user, because this line was missing.
I think there is no reason not to permit this stream - do you agree?

Reply:
If it works, sure